### ISSUE TITLE
Provide deprecated alias for buttonGroup for backwards compatibility

### ISSUE THREAD

**Issue by iangilman** [openseadragon/openseadragon#2146]
As @arthur-flam pointed out in https://github.com/openseadragon/openseadragon/pull/1884#commitcomment-64245151 with 3.0.0 we changed `buttons` to `buttonGroup` without providing a deprecated alias for it. We should fix that for the next release. It should be as simple as providing the following in the prototype of Viewer: 

```
get buttons() {
  console.warn('Viewer.buttons is deprecated; please use Viewer.buttonGroup');
  return this.buttonGroup;
}
```

**Comment by devbyjonah** [comment]
Hello @iangilman , I can try and implement this. I have a question about the package-lock.json file, should I restore it to match the main repo before making a pull request? It was automatically changed when installing dependencies and I wasn't sure if I should include it in the commit or restore the file.

**Comment by iangilman** [comment]
It's preferable not to include the `package-lock.json` changes if you're not updating `package.json`, but I know that the changes do occur. Thank you for cleaning them out of your PR (and thank you for making the PR in the first place!).

**Comment by iangilman** [comment]
This is closed by #2153.
